drivers/media/i2c/irs1125.c

```diff
@@ -29,6 +29,7 @@
 #include <media/v4l2-image-sizes.h>
 #include <media/v4l2-mediabus.h>
 #include <media/v4l2-ctrls.h>
+#include <asm/unaligned.h>
```

Comment:
Includes should normally be in alhpabetical order.

Reply:
Okay, I didn't know that